pyscf/adc/test/test_radc/test_H2O_radc_ea.py

```diff
@@ -52,7 +52,7 @@ def test_ea_adc2(self):
         self.assertAlmostEqual(e, -0.2218560609876961, 6)
```

Comment:
Add analyze() to some properties and tests.

Reply:
Added